icevision/visualize/show_data.py

```diff
@@ -83,6 +84,7 @@ def show_pred(
         display_bbox=display_bbox,
         display_mask=display_mask,
         denormalize_fn=denormalize_fn,
+        **draw_sample_kwargs,
```

Comment:
this is a good catch yeah, once you remove the `class_map` parameter we merge this one =)

Reply:
ok will do